kubeflow/compiled/evaluate_the_model.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: argoproj.io/v1alpha1
 kind: Workflow
 metadata:
   generateName: evaluate-the-model-
-  annotations: {pipelines.kubeflow.org/kfp_sdk_version: 1.0.0, pipelines.kubeflow.org/pipeline_compilation_time: '2020-08-18T15:23:50.716642',
```

Comment:
@bahadir Can you make sure you use `kfp>=1.0.0`. Also, you should avoid compile other functions if they are not changed.